components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -514,5 +520,15 @@ public void setManagementApp(boolean managementApp) {
 
         isManagementApp = managementApp;
     }
+
+    public boolean isB2BSelfServiceApp() {
+
+        return isB2BSelfServiceApp;
+    }
+
+    public void setB2BSelfServiceApp(boolean b2BSelfServiceApp) {
+
+        isB2BSelfServiceApp = b2BSelfServiceApp;
+    }
```

Comment:
Can't we have the same variable name?
```suggestion
    public void setB2BSelfServiceApp(boolean isB2BSelfServiceApp) {

        this.isB2BSelfServiceApp = isB2BSelfServiceApp;
    }
```